mbed_targets/_internal/target_database.py

```diff
@@ -95,9 +83,9 @@ def _response_error_code_to_str(response: requests.Response) -> str:
 def _get_request() -> requests.Response:
     """Make a get request to the API, ensuring the correct headers are set."""
     token = os.getenv(_AUTH_TOKEN_ENV_VAR)
-    if not token:
-        header = None
-    else:
+    if token:
         header = {"Authorization": f"Bearer {token}"}
+    else:
+        header = None
 
     return requests.get(_TARGET_API, headers=header,)
```

Comment:
Could remove the comma here.

Reply:
I have removed it in https://github.com/ARMmbed/mbed-targets/pull/45

`mypy` started failing after the suggestion to invert the logic - that's also fixed in the linked PR

Reply:
Also, this is another reason we should use `black` in the CI.